examples/acceptFromIDs.js

```diff
@@ -0,0 +1,82 @@
+var SteamUser = require('steam-user');
+var client = new SteamUser();
+var TradeOfferManager = require('steam-tradeoffer-manager');
+var config = {
+	"username": "",
+	"password": "",
+	"identitySecret": "",
+	"sharedSecret": "",
+	"acceptSteamIDS": [
+		"SteamID",
+		"SteamID",
+		"SteamID"
+  	]
+};
+var SteamTotp = require('steam-totp');
+var SteamCommunity = require('steamcommunity');
+var community = new SteamCommunity();
```

Comment:
All of the required modules and initializations at the top of the file seem to be very disorderly. Usually it's best to `require` your modules at the top, then initialize them, then add your config.

Also, you'll probably want a description of what this example does at the top (see the link below).

Take a look at the [existing examples](https://github.com/DoctorMcKay/node-steam-tradeoffer-manager/blob/master/examples/storehouse-steam.js) as reference.